openhands/server/session/agent_session.py

```diff
@@ -197,14 +196,24 @@ async def start(
         finally:
             self._starting = False
             success = finished and runtime_connected
-            self.logger.info(
-                'Agent session start',
-                extra={
-                    'signal': 'agent_session_start',
-                    'success': success,
-                    'duration': (time.time() - started_at),
-                },
-            )
+            duration = (time.time() - started_at)
+
+            log_metadata = {
+                'signal': 'agent_session_start',
+                'success': success,
+                'duration': duration,
+                'has_replay': replay_json is not None
```

Comment:
The intention is that has_replay should indicate if this is a new session or resuming one (right?), so we can see if resuming fails more often.

Reply:
replay != resume

replay is a dev feature where you can upload JSON